infra/helper.py

```diff
@@ -16,6 +16,7 @@
 ################################################################################
 
 from __future__ import print_function
+from concurrent.futures import ThreadPoolExecutor
```

Comment:
is this available in python2 without installing any additional packages?

Reply:
Looks like it isn't :( I've tried `apt install python2.7` in the docker image:

```
root@c00fd55d8f02:/# python2.7 -c 'from concurrent.futures import ThreadPoolExecutor'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named concurrent.futures
```

Do you want me to use `threading` instead?

Reply:
https://pypi.org/project/futures/ is a backport from pypi if you can install that

Reply:
thanks @alex, I have it on my machine, so it worked perfectly, but I guess the point is that we don't want users to install that...

Reply:
Ah, indeed. I missed that this was `helper.py` and thus we want it to run cleanly on user's machines.

Maybe time to convert `helper.py` to Python3, where it's included in the stdlib.

Reply:
Let's convert this to python3 later. Could we just use threading for now?

Reply:
Sure, will do!